packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -778,7 +806,7 @@ export class Project extends ProjectBase {
             try {
               parsedArn = stack.parseArn(envVariableValue, ':');
             } catch (e) {}
-            const secretSpecifier: string = parsedArn ? parsedArn.resourceName : envVariableValue;
+            const secretSpecifier: string = parsedArn && parsedArn.resourceName ? parsedArn.resourceName : envVariableValue;
```

Comment:
Why this change?

Reply:
This was a complain by EsLint locally while running the buildup script.

Reply:
Yeah, I would remove this change if the build for the package passes.

Reply:
Actually the build fails for me.